skimage/color/_colorconv.pyx

```diff
@@ -0,0 +1,81 @@
+#cython: boundscheck=False
+#cython: wraparound=False
+#cython: cdivision=True
+
+import cython
+import numpy as np
+from libc.math cimport floor
+
+from .._shared.fused_numerics cimport np_floats
+
+
+def rgb2hsv_inner(np_floats[:, ::] rgb):
+    cdef:
+        np_floats[:, ::] hsv = np.empty_like(rgb)
+        np_floats delta, minv, maxv, tmp
+        Py_ssize_t i, n, ch
+
+    n = rgb.shape[0]
+    for i in range(n):
```

Comment:
What about `prange` here?

Reply:
I can try it. There may not be enough computation for it to provide a benefit, though.

Reply:
I tried `prange` and did see some improvements by a factor of 2-5 for 10 cores (20 threads) on large images. For some reason the `rgb2hsv` function became much slower than before if setting OMP_NUM_THREADS=1 before running the benchmarks though! That seems odd to me, but I wasn't able to quickly diagnose the source of the problem so I have left it without `prange` for now.